src/backends.jl

```diff
@@ -71,4 +71,4 @@ It is a special case of [`ReverseRuleConfigBackend`](@ref).
 !!! note
     To be able to use this backend, you have to load Zygote.
 """
-function ZygoteBackend end
+struct ZygoteBackend <: AbstractReverseMode end
```

Comment:
This also requires changing the docstring and the README which both state that it is just a special `ReverseRuleConfigBackend`.